ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -29,6 +29,12 @@ import { REDESIGN_TRANSACTION_TYPES } from '../../../utils';
 import { getConfirmationSender } from '../utils';
 import { MetaMetricsEventLocation } from '../../../../../../shared/constants/metametrics';
 
+export type OnCancelType = ({
```

Comment:
Minor, this is exactly what I meant but the `Type` is implied, so maybe `OnCancelHandler` or `OnCancelCallback`?

Reply:
done, renamed